src/warnet/deploy.py

```diff
@@ -53,13 +54,60 @@ def deploy(directory):
 
     if (directory / NETWORK_FILE).exists():
         deploy_network(directory)
+        # deploy_fork_observer(directory)
     elif (directory / NAMESPACES_FILE).exists():
         deploy_namespaces(directory)
     else:
         click.echo(
             "Error: Neither network.yaml nor namespaces.yaml found in the specified directory."
         )
 
+def deploy_fork_observer(directory: Path):
+    network_file_path = directory / NETWORK_FILE
```

Comment:
note to future self: this works perfectly fine for a "network per namespace" deployment (i.e., how we would expect people to use warnet in a non-tabconf/wargames setting). however, for tabconf, we will have a network file per team that will deployed into each teams namespace but will want fork observer to be watching all nodes from all namespaces.

as a manual-ish hack, its super easy to just concatenate all of the network files and then use that for fork observers config, but we might also want a fork-observer specific config.yaml file, which would essentially just be a list of nodes with their names/hostnames. 

long-term, I think we will want some python code for managing all of these individual yaml configs to make sure they all line up correctly.